scripts/git-hooks/pre-commit

```diff
@@ -25,7 +25,7 @@ fi
 
 echo "Check that all files conform to Black..."
 invoke black
-if [[ $? != 0 ]]; then
+if [ $? != 0 ]; then
```

Comment:
As an alternate fix, would adding an explicit shebang like `#!/usr/bin/env bash` to the top of the file also work? We really probably shouldn't be trying to write a precommit hook that can run in any/all arbitrary shells the user might be using.

Reply:
Ah, dang GitHub diff view, I missed that there's already a shebang. Sorry for the noise.